reports/apirequests-report-form.twig

```diff
@@ -44,11 +44,14 @@
                 <div class="XiboFilterCustom">
                     <div class="FilterDiv card-body" id="apiRequestsHistoryFilter">
                         <form class="form-inline">
+                            {% set title %}{% trans "Range" %}{% endset %}
+                            {{ inline.dateRangeFilter("reportFilter", title, "", "", "", "", "") }}
```

Comment:
I think the date range from/to date fields should be included in the dateRangeFilter macro automatically, so you do not need to add them separately in the form.

That would mean we can name them accordingly and have more than one range filter per form if we wanted to.

Reply:
Thanks again, Dan! Refactored the code to simply the input handling [here](https://github.com/xibosignage/xibo-cms/pull/2827/commits/8361006629f23de084429d81c2938931a8714076).